modules/microshift-data-cleaning-container-images.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-container-images_{context}"]
+= Cleaning all data and keeping the container images
+
+Retain the {microshift-short} container images while cleaning all data by running the following script. This helps speed up {microshift-short} restart after data cleanup as the necessary container images are already present locally when starting the service. This script performs the following cleanup actions:
+
+* Stop and disable {microshift-short} services
+* Delete all {microshift-short} pods
+* Reset network configuration
+* Delete the /var/lib/microshift data directory
+* Delete OVN configuration
+
+.Procedure
+
+. Run the following command.
```

Comment:
same comment here; we need to adhere to docs style

Reply:
done